src/message/Message.story.mdx

```diff
@@ -0,0 +1,89 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+import { text } from '@storybook/addon-knobs'
+import {Fragment} from 'react'
+
+import { Message } from './index'
+
+<Meta title="Widgets|Message" />
+
+# ** Message **
+
+## Basic message
+
+<Preview>
+  <Story name="Default">
+    <Message active={false}>{text('text', 'A simple message')}</Message>
+  </Story>
+</Preview>
+
+## Conversation
+
+<Preview>
+  <Story name="Conversation">
+    <Fragment>
```

Comment:
Lots of messages, it would be better to separate on "variations", Check `ButtonGroup` to an example